docs/guides/universal-profile/transfer-lyx.md

```diff
@@ -27,20 +27,35 @@ The parameters of the function will be as follow:
 
 Since we are just making a simple LYX transfer, the fourth parameter `_data` will be empty.
 
+## Setup
+
+To complete this mini-guide, we will need two things:
+
+- the ABIs of the `UniversalProfile` and `KeyManager` contracts.
+- the address of our Universal Profile we want to send LYX from.
+
+```shell
+npm install web3 @lukso/lsp-smart-contracts --save
```

Comment:
`--save` is not needed

Reply:
I have removed it, but the other guides mention it